zenpacklib.py

```diff
@@ -6315,6 +6315,13 @@ def create_zenpack_srcdir(zenpack_name):
     import shutil
     import errno
 
+    # validate ZP name - it must begins with "ZenPacks.*"
+    if not zenpack_name.startswith('ZenPacks.'):
+        sys.exit(
+            "`{}` is a wrong name for ZenPack - zenpack name must starts "
+            "with `ZenPacks`. Please, check ZP name.".format(zenpack_name)
+        )
```

Comment:
I would suggest changing message to something like this:
"ZenPack name is incorrect, it does not match ZenPack naming requirenments. Please see documentation: https://zenpack-sdk.zenoss.com/en/stable/managing-zenpacks-4.html#naming-a-zenpack"